<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PalROEditor</class>
 <widget class="QMainWindow" name="PalROEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PalROEditor</string>
  </property>
  <widget class="QWidget" name="mainLayaout">
   <layout class="QHBoxLayout" name="main_layaout" stretch="1,2">
    <item>
     <layout class="QVBoxLayout" name="sprite_sel">
      <item>
       <widget class="QGraphicsView" name="spritePreview"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="jobSel">
          <item>
           <property name="text">
            <string>Novice</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="genderSel">
          <item>
           <property name="text">
            <string>M</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>F</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="Action"/>
        </item>
        <item>
         <widget class="QSpinBox" name="Hair_id">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Hair">
         <attribute name="title">
          <string>Hair</string>
         </attribute>
         <widget class="QListWidget" name="palListHairW">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>471</width>
            <height>231</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="Body">
         <attribute name="title">
          <string>Body</string>
         </attribute>
         <widget class="QListWidget" name="palListBodyW">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>471</width>
            <height>231</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,1,5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,20">
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="text">
           <string>ðŸ’¾</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pal_path_label">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="o_palette"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="o2n">
          <property name="text">
           <string>â†“</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="n2o">
          <property name="text">
           <string>â†‘</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="o2n_full">
          <property name="text">
           <string>â†¡</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="n2o_full">
          <property name="text">
           <string>â†Ÿ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="n_palette"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
